manager/csi/plugin.go

```diff
@@ -204,8 +207,13 @@ func (p *plugin) PublishVolume(ctx context.Context, v *api.Volume, nodeID string
 	if !p.publisher {
 		return nil, nil
 	}
+	csiNodeID := p.swarmToCSI[nodeID]
+	if csiNodeID == "" {
+		log.L.Errorf("CSI node ID is not yet registered. Plugin: %s , Swarm node ID: %s", p.name, nodeID)
```

Comment:
should we maybe change the error message to something less specific here?

"CSI node ID not found for given Swarm node ID. Plugin: %s , Swarm node ID: %s"

or something? Then this error message will not be misleading if we run into it in the future.